admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -10,50 +10,53 @@ module.exports = {
-    order_delivery: './js/pages/order/delivery',
-    product_preferences: './js/pages/product-preferences',
-    imports: './js/pages/import',
-    localization: './js/pages/localization',
-    invoices: './js/pages/invoices',
-    geolocation: './js/pages/geolocation',
-    payment_preferences: './js/pages/payment-preferences',
-    email: './js/pages/email',
-    sql_manager: './js/pages/sql-manager',
     catalog_product: './js/pages/catalog/product',
-    backup: './js/pages/backup',
     categories: './js/pages/categories',
-    module_card: './js/app/pages/module-card',
-    translation_settings: './js/pages/translation-settings',
-    webservice: './js/pages/webservice',
-    module: './js/pages/module',
-    meta: './js/pages/meta',
+    cldr: './js/app/cldr',
+    cms_page: './js/pages/cms-page',
+    cms_page_form: './js/pages/cms-page/form',
     contacts: './js/pages/contacts',
+    currency: './js/pages/currency',
+    customer: './js/pages/customer',
+    email: './js/pages/email',
     employee: './js/pages/employee/index',
     employee_form: './js/pages/employee/form',
-    customer: './js/pages/customer',
+    form_popover_error: './js/components/form/form-popover-error',
+    geolocation: './js/pages/geolocation',
+    imports: './js/pages/import',
+    improve_design_positions: './js/pages/improve/design_positions',
+    invoices: './js/pages/invoices',
     language: './js/pages/language',
+    localization: './js/pages/localization',
+    logs: './js/pages/logs',
+    main: './js/theme.js',
+    manufacturer: './js/pages/manufacturer',
+    manufacturer_address_form: './js/pages/manufacturer/manufacturer_address_form.js',
+    meta: './js/pages/meta',
+    module: './js/pages/module',
+    module_card: './js/app/pages/module-card',
+    order_delivery: './js/pages/order/delivery',
+    order_preferences: './js/pages/order-preferences',
+    payment_preferences: './js/pages/payment-preferences',
     product_page: './js/product-page/index',
-    currency: './js/pages/currency',
-    supplier: './js/pages/supplier',
-    themes: './js/pages/themes',
+    product_preferences: './js/pages/product-preferences',
     profiles: './js/pages/profiles',
+    sql_manager: './js/pages/sql-manager',
+    stock: './js/app/pages/stock',
+    supplier: './js/pages/supplier',
     tax: './js/pages/tax',
-    cms_page: './js/pages/cms-page',
-    cms_page_form: './js/pages/cms-page/form',
-    form_popover_error: './js/components/form/form-popover-error',
-    manufacturer: './js/pages/manufacturer',
-    manufacturer_address_form: './js/pages/manufacturer/manufacturer_address_form.js',
+    themes: './js/pages/themes',
+    translation_settings: './js/pages/translation-settings',
+    translations: './js/app/pages/translations',
+    webservice: './js/pages/webservice',
   },
   output: {
     path: path.resolve(__dirname, '../public'),
     filename: '[name].bundle.js',
+    libraryTarget: 'window',
+    library: '[name]',
```

Comment:
allow to use each `entry` as `window.[name]`, for example: `window.cldr.NumberFormatter`